package/osx/scripts/merge-electron.mjs

```diff
@@ -0,0 +1,26 @@
+
+import 'process';
+import { execSync } from 'child_process';
+import { makeUniversalApp } from '@electron/universal';
```

Comment:
@matheustavaresdev, what's the right way to run this script in a way that ensures `@electron/universal` is available? I'm executing it in CMake here:

https://github.com/rstudio/rstudio/blob/6358e490e8ac73e10af12af2a76f50e76d0d8243/package/osx/cmake/merge-electron.cmake#L21-L30

Would it suffice to just call `npm install @electron/universal` before running this script?